### PR TITLE
Fix timeouts related to metalearnings tests

### LINKED ISSUE #1506: Fix timeouts related to metalearnings tests

### PR DESCRIPTION
**Pull request by eddiebergman**
Adding debug statements for now to find the problem on the github action servers as it cleanly finished locally.

Closes #1506

### CODE REVIEW

**Comment by eddiebergman** [comment]
I switched to using `import tmpfile and using `gettempdir()` instead of explicitly using `/tmp`. It seemed to not trigger as a result. Maybe the github action servers have some special folder for it and the stochastic nature of the problem was due to node allocation and usage.

I'm going to run the actions a few time and see if it comes up again.

File: test/test_scripts/test_metadata_generation.py [hunk 1]
```diff
@@ -174,12 +188,20 @@ def test_metadata_generation(self):
             self.working_directory,
         )
         print("COMMAND: %s" % cmd)
+
+        # With a timeout of 30seconds, it seems to not complete here on the github
+        # action servers. It's stochastic and
```
Comment (mfeurer) [inline test/test_scripts/test_metadata_generation.py:193]: and?

**Review by mfeurer — COMMENTED** [review]
Looks good to me. Does it solve all issues?

**Comment by eddiebergman** [comment]
I just set explicit timeouts for each step and bumped the overall timeout per test from `300s` to `600s`. If one of them actually does hang then at least we'll get a debug message in the future

**Comment by eddiebergman** [comment]
I removed the comment you left review on, if all the tests pass then I think that can be merged

**Pull request merged**